examples/index.html

```diff
@@ -400,6 +400,21 @@ <h2 class="bg-dark">Interaction</h2>
                         </div>
                     </a>
                 </div>
+                <div class="col-4 p-2">
+                    <a href="./interaction/call_activity_with_tabs_on_click/index.html" target="_blank" class="card-link">
+                        <div class="card p-2">
+                            <div class="card-header">
+                                <div class="card-title">Call activity with tabs</div>
+                                <div class="card-subtitle"></div>
+                            </div>
+                            <div class="card-image">
+                                <img src="static/img/preview/interaction/preview_call_activity_with_tabs_on_click.png" class="img-responsive" alt="call activity with tabs">
+                            </div>
+                            <div class="card-body">Switch BPMN diagram display on click on call activity</div>
```

Comment:
:warning: seems not consistent with the title in the page
:question: can we find a wording more explicit? When I first saw 'tab', I thought that the call activity diagram would open in a new Browser tab, not in a tab of the page

Reply:
Is it better now ? Or do you have other idea ? 🙂

Reply:
Yep :smile_cat: